omnibus/config/projects/iot-agent.rb

```diff
@@ -84,10 +84,9 @@
   COMPRESSION_THREADS = 1
 end
 
-# On armv7, dpkg is built as a 32bits application, which means
-# we can only address 32 bits of memory, which is likely to OOM
-# if we use too many compression threads or a too agressive level
-if ENV.has_key?("DEPLOY_AGENT") && ENV["DEPLOY_AGENT"] == "true" && (!ENV.has_key?("PACKAGE_ARCH") || ENV["PACKAGE_ARCH"] != "armhf")
+if ENV.has_key('FORCED_COMPRESSION_LEVEL')
```

Comment:
I think I'd prefer a name such as this but not blocking

```suggestion
if ENV.has_key('PACKAGE_COMPRESSION_LEVEL')
```